Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -190,6 +192,15 @@ static bool isCopyDisplayedPixelBufferAvailable()
 }
 #endif
 
+void MediaPlayerPrivateWebM::responseReceived(PlatformMediaResource& resource, const ResourceResponse& response)
+{
+    ALWAYS_LOG(LOGIDENTIFIER);
+    
+    m_origins.add(SecurityOrigin::create(response.url()));
+    m_resolvedOrigin = SecurityOrigin::create(response.url());
```

Comment:
swap the two and add m_resolvedOrigin to m_origins.
Will save a memory allocation

Reply:
On second thought, I don't think we need to keep track of the multiple origins.
We only care if we failed CORS along the way, see below